webui-desktop

```diff
@@ -89,6 +89,26 @@ setup_user_environment() {
     echo "$INSTALLER_USER"
 }
 
+get_default_browser() {
+    # Read the default browser from the anaconda runtime configuration file but wait till it's available
+    # FIXME: Related to: https://issues.redhat.com/browse/INSTALLER-4223
+    while [ ! -f /run/anaconda/anaconda.conf ]; do
+        echo "Waiting for anaconda.conf to be available..." >&2
+        sleep 1
```

Comment:
Maybe we can add a timeout there ? Maybe we could spawn a dialog telling the user that backend failed to start in time ? Just some followup PR ideas. :)